introduction.md

```diff
@@ -0,0 +1,4 @@
+Go to the Deadly Vipers organization on GitHub and look for the "dojo_rules" repository. Follow the tasks below to begin working with this repository.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.